### PR TITLE
Feature: levelbuilder: Add missing blocks to droplet palette using a drop down

### PR DESCRIPTION
**Pull request by pcardune**
What it looks like when you first create a level:

image

Now you delete some blocks from the json config:

image

Or you edit the json to be invalid syntax:

image

cc @joshlory @Hamms

### CODE REVIEW

**Comment by pcardune** [comment]
This is for this axosoft task btw: https://codeorg.axosoft.com/viewitem?id=651&type=features&force_use_number=true

**Comment by poorvasingal** [comment]
Nice!  Thanks Paul :)

File: apps/test/unit/lib/levelbuilder/DropletPaletteSelectorTest.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    });
+
+    describe("When selecting an item", () => {
+
+      it("updates the editor to include that value", () => {
+        const selector = mount(<DropletPaletteSelector editor={editor} palette={{a: null, b: null}} />);
+        selector.find('select').props().onChange({target: {value: 'b'}});
+        expect(editor.getValue()).to.equal(`{
+  "b": null
+}`);
```
Comment (islemaster) [inline apps/test/unit/lib/levelbuilder/DropletPaletteSelectorTest.js:88]: Weird whitespace?

File: apps/test/unit/lib/levelbuilder/DropletPaletteSelectorTest.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        expect(selector.containsMatchingElement(
+          <select disabled>
+            <option>All blocks have been added</option>
+          </select>
+        )).to.be.true;
+
+        expect(editor.getValue()).to.equal(`{
+  "b": null,
+  "a": null
+}`);
```
Comment (islemaster) [inline apps/test/unit/lib/levelbuilder/DropletPaletteSelectorTest.js:107]: Weird whitespace?

**Pull request merged**